WI #557: Normative Requirements Refinement - Part 2 (#798)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/consumedunit.md

The ConsumedUnit column adheres to the following requirements:

* ConsumedUnit MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports the measurement of usage.
* ConsumedUnit MUST be of type String.
* ConsumedUnit MUST conform to [StringHandling](#stringhandling) requirements.
* ConsumedUnit SHOULD conform to [UnitFormat](#unitformat) requirements.
* ConsumedUnit nullability is defined as follows:

>   * ConsumedUnit MUST be null when [ChargeCategory](#chargecategory) is not "Usage", or when ChargeCategory is "Usage" and [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Unused".
>   * When ChargeCategory is "Usage" and CommitmentDiscountStatus is not "Unused", ConsumedUnit adheres to the following additional requirements:
>     * ConsumedUnit MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction".
>     * ConsumedUnit MAY be null when ChargeClass is "Correction".

Author: refer to [this conversation ](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/798/files#r2001081250) for reasoning and 

```suggestion
  * ConsumedUnit MUST be null when ConsumedQuantity is null.
  * ConsumedUnit MUST NOT be null when ConsumedQuantity is not null.
```

Author: In addition to here provided thumbs-up, since this was already discussed and approved by TF-1, I’m committing the suggested change and resolving this conversation.
